Kinvey/Kinvey/FileStore.swift

```diff
@@ -98,12 +98,20 @@ open class FileStore<FileType: File> {
     
     /// Uploads a `UIImage` in a PNG or JPEG format.
     @discardableResult
-    open func upload(_ file: FileType, image: NSImage, imageRepresentation: ImageRepresentation = .png, ttl: TTL? = nil, completionHandler: FileCompletionHandler? = nil) -> Request {
+    open func upload(
+        _ file: FileType,
+        image: NSImage,
+        imageRepresentation: ImageRepresentation = .png,
+        ttl: TTL? = nil,
```

Comment:
totally agree, all those changes will also help to solve this problem here: https://hub.codebeat.co/docs/software-quality-metrics#arity